FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/V2DemoListActivity.kt

```diff
@@ -520,6 +524,12 @@ class V2DemoListActivity : ComponentActivity() {
                                     trailingAccessoryContent = if (it.badge != Badge.None) {
                                         {
                                             Badge(
+                                                modifier = when(it.badge) {
+                                                    Badge.New -> Modifier.background(brush = SolidColor(FluentTheme.aliasTokens.errorAndStatusColor[FluentAliasTokens.ErrorAndStatusColorTokens.SevereForeground2].value()), shape = RoundedCornerShape(100.dp))
```

Comment:
is the brush colors approved by design team or we decided to have these for modifiers indicators ?

Reply:
Not sure if there is a someone actively looking at this design. Instead we can come up with some good color specifications.